.github/workflows/create-tag-from-mix-version.yaml

```diff
@@ -0,0 +1,36 @@
+name: create-tag-from-mix-version
+on:
+  workflow_call:
+    inputs:
+      runs-on:
+        required: false
+        type: string
+        default: ubuntu-latest
+jobs:
+  create-tag-from-mix-file:
+    name: Create Tag from Mix Version
+    if: github.event.pull_request.merged == true
+    runs-on: ${{ inputs.runs-on }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up git
+        id: git_setup
```

Comment:
We don't use an `id` unless the step needs to be referred to, e.g. because the step has outputs.